frontend/src/components/pages/Dashboard/components/DropoffCard.tsx

```diff
@@ -0,0 +1,116 @@
+import { StarIcon, TimeIcon } from "@chakra-ui/icons";
+import {
+  Badge,
+  Box,
+  HStack,
+  IconButton,
+  Menu,
+  MenuButton,
+  MenuItem,
+  MenuList,
+  Text,
+  useDisclosure,
+} from "@chakra-ui/react";
+import React from "react";
+import { useHistory } from "react-router-dom";
+
+import * as Routes from "../../../../constants/Routes";
+import { EllipsisIcon } from "../../../common/icons";
+import { SchedulingFormProps } from "../../Scheduling/types";
+import DeleteScheduleModal from "./DeleteScheduleModal";
+
+interface DropoffCardProps {
+  schedule: SchedulingFormProps;
+  onDelete: () => void;
+}
+
+const DropoffCard = ({ schedule, onDelete }: DropoffCardProps): JSX.Element => {
+  const startDate = new Date(schedule.startTime).toDateString();
+  const startTime = new Date(schedule.startTime).toLocaleTimeString();
+  const history = useHistory();
+
+  const { isOpen, onOpen, onClose } = useDisclosure();
+
+  return (
+    <Box
+      maxW="sm"
+      mb="24px"
+      border="1px solid"
+      borderColor="hubbard.100"
+      borderRadius="8px"
+    >
+      <Box pl="6" pr="6" pb="6" pt="4">
+        <HStack spacing="10">
```

Comment:
I believe if you touch anywhere in the card, it should also lead to the edit page, so I guess add the `onClick` function for box as well?

Reply:
Are you sure? What's the purpose of the menu button for edit then?

Reply:
I just made it clickable then.